### PR TITLE
Support SLE-15

### PR DESCRIPTION
**Pull request by smithfarm**
Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -23,4 +23,4 @@ python-keystoneclient==3.5.0
 python-neutronclient==4.1.1
 python-novaclient==6.0.0
 python-openstackclient==3.2.0
-requests==2.18.1
```
Comment (kshtsk) [inline requirements.txt]: Is this related to the sle15?

**Comment by smithfarm** [comment]
@kshtsk The requirements.txt commit obviously is unrelated to the "theme" of this PR, but appears to be needed (otherwise I cannot deploy teuthology-openstack anymore).
